### PR TITLE
Close #126 : Docker image for pytoutv CLI

### LINKED ISSUE #126: pytoutv in a Docker image
Hello, 
maybe an issue is not the right place, but I've packaged pytoutv in a Docker container, see : https://github.com/anthonydahanne/toutv for examples and how to use it.

It should be updated frequently (the image is available from : https://hub.docker.com/r/anthonydahanne/toutv/ ) and if you want to use the Dockerfile and push the image to your own Github repository, let me know.

I can also help with it all - je suis sur Montréal, autour d'une bière çà me ferait plaisir d'en parler !

### PR DESCRIPTION
**Pull request by anthonydahanne**

### CODE REVIEW

**Comment by anthonydahanne** [comment]
CI failed for something not related to my changes !

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# toutv in a Docker container
+It's just a Docker image for https://github.com/bvanheu/pytoutv
+
+## How to use
+Very similarly to the pytoutv client, except you don't have to install python3 nor pip3 nor toutv scripts and their dependencies, everything is provided in this image.
+
+Here are a few examples on how to use this image , [list of commmands is provided on pytoutv Github repo](https://github.com/bvanheu/pytoutv) : 
+
+```
+$ docker run anthonydahanne/toutv
```
Comment (simark) [inline docker/README.md]: That example would have to change if you address the comment below.

**Review by simark — CHANGES_REQUESTED** [review]
Cool, it works nicely.  I have some minor comments about the README.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Docker image definition for https://github.com/bvanheu/pytoutv
+FROM python:3.7-stretch
+LABEL maintainers="Anthony Dahanne <anthony.dahanne@gmail.com>"
+RUN pip3 install pytoutv
+VOLUME /media
+WORKDIR /media
+ENTRYPOINT ["toutv"]
+CMD ["-V"]
```
[the remark this answers is not in the document]
Comment (anthonydahanne) [reply docker/Dockerfile]: ok

File: docker/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Infos:
+
+  * Network: CBFT
+```  
+
+Also, you can mount a local directory to the /media volume; in other words, you can download files to your host filesystem :
+
+```
+$ docker run -v /Users/anthony/Downloads/:/media/  anthonydahanne/toutv fetch -q MAX INFOMAN S19E01
```
[the remark this answers is not in the document]
Comment (anthonydahanne) [reply docker/README.md]: cool, ok

File: docker/README.md (hunk 1 above)
Comment (anthonydahanne) [reply docker/README.md]: true

**Comment by simark** [comment]
Thanks.  Could you please squash your commits in a single one?

**Comment by anthonydahanne** [comment]
ok

**Comment by anthonydahanne** [comment]
done

**Comment by simark** [comment]
I already have a dockerhub account, do I need to make a new one just for this?

Alternatively, would it work if I gave you some permissions on this repo?

**Comment by anthonydahanne** [comment]
> I already have a dockerhub account, do I need to make a new one just for this?
If you use your personal github account, you'll have to create a simark/toutv dockerhub repo;  actually I'm not even sure you can; it all comes down to how dockerhub links to github accounts

> Alternatively, would it work if I gave you some permissions on this repo, so you can retain the ownership of the docker repository?

sure, but I don't know what kind of permissions are required.

I'd think having an org would help, but maybe we can work something with personal accounts and permissions - we could chat about it on hangout, I'm anthony point dahanne arobase gmail point com

**Comment by simark** [comment]
Sorry I took a bit of time to come back to this.

I created a pytoutv org with a pytoutv repository and added you as an owner:

https://hub.docker.com/u/pytoutv/dashboard/

**Comment by simark** [comment]
I uploaded a build tagged v3.0.2, can you check if it works for you?

**Comment by anthonydahanne** [comment]
hello @simark ; you created a repository that is not an automated build.
It needs to be an automated build, so that it will be build automatically by Docker Hub infra after any new commits.
Please delete this repository, and create a new "automated build", then configure it so that it picks up the Dockerfile under the docker/ folder
Here is a screenshot, let me know if oyu have any questions (you gave eme rights on Dockerhub, but I can't link to github.com/bvanheu since I don't have org membership there; but anyway, you're almost there !)
screen shot 2018-10-01 at 12 46 52

**Comment by simark** [comment]
Hmm, even I can only create a build based on my repo (simark/pytoutv).  Either @bvanheu needs to create it, or we create a pytoutv github organization and move the official pytoutv repo there.  What would you prefer, @bvanheu ?

**Comment by anthonydahanne** [comment]
@simark @bvanheu  this dockerhub / github integration seems to prefer an organization - what about a pytoutv organization on Github ? that could help setting up permissions and other things too

**Pull request merged**